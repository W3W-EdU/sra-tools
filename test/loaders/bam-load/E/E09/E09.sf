#bam-load error test #09 : a pair of alignments have each both FIRST and LAST flag set

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E09.fasta

# write the final SAM-output into this file
sam-out:E09.sam

# write a config-file for bam-load
config:E09.config

p:name=A1,pos=100,tag=T1
p:name=A1,pos=300,tag=T2
t:name=T1,action=set,flag=LAST
t:name=T2,action=set,flag=FIRST

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50
