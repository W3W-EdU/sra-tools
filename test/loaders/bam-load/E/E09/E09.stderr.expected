warn: Spots with more than two fragments have been encountered
